indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -659,9 +663,14 @@
         _canActOn: function(category) {
             var self = this;
             var ids = self.options.actionOn.categories.ids;
-            return !(self.options.actionOn.categoriesWithSubcategories.disabled && category.deep_category_count) &&
-                   !(self.options.actionOn.categoriesWithEvents.disabled && category.has_events) &&
-                   !(self.options.actionOn.categories.disabled && _.contains(ids, category.id));
+            var categoriesWithSubcategories = self.options.actionOn.categoriesWithSubcategories;
+            var categoriesWithEvents = self.options.actionOn.categoriesWithEvents;
+
+            return !(categoriesWithSubcategories.disabled && category.deep_category_count) &&
+                   !(categoriesWithEvents.disabled && category.has_events) &&
+                   !(self.options.actionOn.categories.disabled && _.contains(ids, category.id)) ||
+                   (categoriesWithEvents.disabled && category.has_events &&                 // Make sure hubrid categories are treated
+                    !categoriesWithSubcategories.disabled && category.deep_category_count); // as categories with subcategories
```

Comment:
true, i somehow thought we were specifying where we can perform an action ("in categories with events", "in categories with subcategories")